### DISCUSSION TITLE
Сделать просмотр для файлов PDF

### DISCUSSION THREAD

**Discussion by a1ext0** [totumonline/totum-ru-issues-and-discussions#341] [Как это сделать?] [answered]
Было бы здорово, если бы можно было выводить на просмотр файлы загруженные в формате PDF. Сейчас, для того чтобы их просмотреть, приходится скачивать файл и открывать на компьютере. Насколько я знаю большинство браузеров поддерживают открытие PDF прямо внутри себя (Chrome и Firefox точно). Возможно есть уже какой-то вариант, но в документации не видел пока 😁

**Comment by ttm-alexeypolunin** [comment, marked as answer]
В иконку файла надо ткнуть. Но вообще надо добавить кнопку `показать` в панели по правой кнопке. Мы это сделаем через месяц-два, когда будет готов функционал, позволяющий открывать на редактирование .docx, .xlsx прямо из поля Файл в браузере.
